server/game/cards/13.3-PoS/MaesterMullin.js

```diff
@@ -0,0 +1,47 @@
+const DrawCard = require('../../drawcard.js');
+const ChallengeTypes = require('../../ChallengeTypes');
+
+class MeasterMullin extends DrawCard {
+    setupCardAbilities() {
+        this.action({
+            title: 'Kneel to choose a character',
+            when: {
+                onCardEntersPlay: event => event.card === this
+            },
+            target: {
+                cardCondition: card => card.location === 'play area' && card.getType() === 'character'
+            },
+            handler: (context) => {
+                this.selectedCharacter = context.target;
+
+                this.game.promptWithMenu(this.controller, this, {
```

Comment:
There's also an explicit "promptWithIcon" method, but this works.

Reply:
Are you sure? I can't find a method with that name, I just copied calm over westeros